app/merginapi.cpp

```diff
@@ -261,6 +309,41 @@ void MerginApi::saveAuthData()
     settings.endGroup();
 }
 
+void MerginApi::createProjectFinished()
+{
+    QNetworkReply* r = qobject_cast<QNetworkReply*>(sender());
+    Q_ASSERT(r);
+
+    if (r->error() == QNetworkReply::NoError)
+    {
+        emit notify("Project created");
+        emit projectCreated();
```

Comment:
Would it make sense to pass project name in the signal as well to make it consistent with delete project signal?

Reply:
Additionally, it would be nice to connect it with api's project list and have newly created project listed there without fetching a list again (Yes, similarly to delete project)

Reply:
yeah but at this point we don't want to have create/delete project functionality in Input - something people should do in web client / qgis plugin...